basic-examples/bind-constants/workflow.yaml

```diff
@@ -0,0 +1,43 @@
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        repeated_inputs:
+          type:
+            type_id: ref
+            id: RepeatedValues
+        iterations:
+          type:
+            type_id: list
+            items:
+              id: SubRootObject
+              type_id: ref

```

Comment:
Are you sure you want to reference schemas in the parent workflow instead of referencing the ones in the subworkflow?

Reply:
am I allowed to use namespaces?

Reply:
used namespaces